ocaml/typing/typecore.ml

```diff
@@ -3520,7 +3520,7 @@ let check_recursive_class_bindings env ids exprs =
 
 
 (* Is the return value annotated with "local_" *)
-let is_local_returning_expr, is_local_returning_case_rhs =
+let is_local_returning_expr, is_local_returning_case =
```

Comment:
I don't agree with the rename from `case_rhs` to `case` here &mdash; there is still a well-defined notion of "case rhs" on the AST, it's just that the embedding is a bit more complicated.

Reply:
Resolved by the below